### ISSUE TITLE
Out-of-tree builds broken. Can't find embedwast.py

### ISSUE THREAD

**Issue by sbc100** [WebAssembly/binaryen#1568]
Looks like a recent cmake change broke out of tree builds:

```
binaryen-out) $ ninja
[0/1] Re-running CMake...
-- No build type selected, default to Release
-- Building with -std=c++11
-- Building with -Wall
-- Building with -Werror
-- Building with -Wextra
-- Building with -Wno-unused-parameter
-- Building with -fno-omit-frame-pointer
-- Building with -fPIC
-- Building with -O2
-- Building with -UNDEBUG
-- Building with -fcolor-diagnostics
-- Configuring done
-- Generating done
-- Build files have been written to: /usr/local/google/home/sbc/dev/wasm/binaryen-out
[1/56] Generating WasmIntrinsics.cpp
FAILED: src/passes/WasmIntrinsics.cpp 
cd /usr/local/google/home/sbc/dev/wasm/binaryen-out/src/passes && python ../../scripts/embedwast.py /usr/local/google/home/sbc/dev/wasm/binaryen-out/src/passes/WasmIntrinsics.cpp
python: can't open file '../../scripts/embedwast.py': [Errno 2] No such file or directory
[7/56] Building CXX object src/wasm/CMakeFiles/wasm.dir/wasm-binary.cpp.o
ninja: build stopped: subcommand failed.
```

**Comment by kripken** [comment]
Was this fixed by that PR?

**Comment by sbc100** [comment]
Fixed in #1569
